pJanks/sour-gazpacho
# Testing async functionality

#### What's this PR do?
This adds tests for adding and removing ratings. It also makes minor adjustments to some components for testing purposes and adds PropTypes to necessary components.

#### Where should the reviewer start?
MovieDetails.test.js

#### How should this be manually tested?
Pull down the branch and check for broken or buggy behavior and run tests to ensure they are passing.

#### Any background context you want to provide?
I believe there is still one test that I cannot get to pass. Perhaps we can figure it out together.

#### What are the relevant tickets?
#9 

#### Screenshots (if appropriate)
none.

#### Questions:
none.
